### DISCUSSION TITLE
样式混乱！官网给的脚本根本不包含在依赖？！？？

### DISCUSSION THREAD

**Discussion by summer-077** [bytedance/xgplayer#1166] [Q&A]
下载完成时候使用的时候样式是混乱的，而且也没有包含文档给的eject脚本，根本无法运行npx命令。

为了排除我自己的问题，我特地找了一个运行出来的项目，他的运行出来样式正确
![image](https://github.com/bytedance/xgplayer/assets/118544684/8cee9db9-e947-4ea2-974b-9df1f0a982dd)

他的依赖文件如下，且能找到eject脚本：
![image](https://github.com/bytedance/xgplayer/assets/118544684/8cd275ae-0c53-41e0-8a4e-ccd99d4dccb6)

我卸载依赖之后，重新安装，全局文件搜索没有找到eject脚本，那么官网给运行命令就有问题
![image](https://github.com/bytedance/xgplayer/assets/118544684/ebbc440a-0c67-46a9-b411-e2c2fc853be5)

且我重新运行项目出现样式混乱问题
![image](https://github.com/bytedance/xgplayer/assets/118544684/d2ce735d-1d09-4a0e-b22b-fa9177165aee)

各位佬有出现这样的问题吗

**Comment by Johnwikix** [comment]
一样

**Comment by wangshangluobai** [comment]
import "xgplayer/dist/index.min.css"
